client/client-multi/client-c/build.sh

```diff
@@ -168,7 +108,10 @@ case "$cmd" in
         UNITTESTS_COMPILE=1
```

Comment:
It seems convenient to me having the ability to not compile unit tests. Here is why: I'm implementing a feature and finally I want to check whether SDK is compiled. I know that I broke unit tests (and will definitely fix them), but for now I want to check whether SDK is compiled. In case if unit tests are compiled, i'll be made to scroll all the stack trace related to unit tests build failure to see whether sdk is compiled. @forGGe what do you think?